src/odemis/gui/cont/temperature.py

```diff
@@ -27,10 +27,31 @@
 import numpy
 import time
 import logging
+from odemis.gui.util import call_in_wx_main
 import odemis.gui as gui
-from odemis.gui import comp
+from odemis.gui.comp import popup
 import odemis.model as md
 
+MIN_PERIOD_WARNING = 60
+
+
+@call_in_wx_main
+def set_label_colour(label, colour, bold=False):
+    """
+    Set a wx.StaticText foreground colour and font style to bold/unbold
+    label: wx.StaticText object
+    colour: wx.Colour object
+    bold: (bool) true to set to bold
+    """
+
+    label.SetForegroundColour(colour)
+    if bold:
+        font = label.GetFont()
```

Comment:
You can move this line out of the if block.
You'd recommend also putting the .SetFont() afterwards.